run--01a--prep-geo-data.Rmd

```diff
@@ -165,22 +165,42 @@ try(zctaShp   <-    zctas(cb = TRUE, year = tiger_year, starts_with = zcta_start
   # the zctas() function can only return by state for years 2000 and 2010 
 
 # Download school district file from Census using tigris package
-elShp <- school_districts(state = {my_state_fip}, type = 'elementary', year = tiger_year)
-scShp <- school_districts(state = {my_state_fip}, type = 'secondary',  year = tiger_year)
-unShp <- school_districts(state = {my_state_fip}, type = 'unified',    year = tiger_year)
+elShp <- school_districts(state = {my_state_fip}, type = 'elementary', year = tiger_year + 1)
```

Comment:
The pull using `school_districts()` will fail if the specified year is not available. Unless there's some way to ensure that `tiger_year + 1` is available (it doesn't seem clear to me that it's assured based on context), I would suggest using a `try()` statement to attempt a pull using `tiger_year + 1`, and then search for successively earlier years if the `try()` returns a fail code.